tpm2/tpm2_test.go

```diff
@@ -41,7 +41,12 @@ var (
 	mssimPlatformAddr = flag.String("mssim_platform_addr", "localhost:2322", "Host and port of the simulator's platform listener")
 )
 
-func openTPM(t *testing.T) io.ReadWriteCloser {
+type T interface {
```

Comment:
use `testing.TB` instead

Reply:
I knew there had to be a type for this but didn't find it (also because TB does not have its functions listed in godoc's TOC for testing).